odl/operator/operator.py

```diff
@@ -470,10 +470,10 @@ def __init__(self, domain, range, linear=False):
             raise TypeError('`range` {!r} is not a `Set` instance'
```

Comment:
How about the guys in `__new__`? Or do we consider them part of the subclass API?